### ISSUE TITLE
Макет

### ISSUE THREAD

**Issue by ZvezdocheTag** [RusRogo/RusRogo.github.io#3]
Верстка слайдера:

 `overlay` вынесен за пределы блока которому он привязан, плохо из-за этого страдает верстка, текст вываливается из overlay очень не ровно...

На логотип hover эффект underline не нужно, логотип зачастую кликабельный но без эффектов.

**Comment by ZvezdocheTag** [comment]
Секция `class=news` , там ты используешь margin-bottom это ок, а потом margin-top задаешь в дочернем элементе `class=news__item`, непонятно

**Comment by RusRogo** [comment]
> overlay вынес потому, что он в макете на всю ширину, как и фон для каждого блока (

> wrapper

 в адаптиве)
![home_768px](https://user-images.githubusercontent.com/42811613/47257136-bca45480-d492-11e8-866a-0eb41a388042.jpg)
как

**Comment by ZvezdocheTag** [comment]
> `overlay` вынес потому, что он в макете на всю ширину, как и фон для каждого блока (`wrapper` в адаптиве)
> Как-то можно по-другому?
> ![home_768px](https://user-images.githubusercontent.com/42811613/47257136-bca45480-d492-11e8-866a-0eb41a388042.jpg)

Да, overlay принадлежит блоку `wrapSlider`, соответственно он должен там распологаться. Это можно реализовать с помощью псевдоэлементов `:before, :after`

**Comment by ZvezdocheTag** [comment]
> Логотип исправил
> `class=news` исправил
> Я еще одну вещь понял - адаптив верстать нужно - каждый блок во всех разрешениях сразу.
> А не сверстать (как я) макет полностью на 320px, потом на 480px - всю конструкцию блоков переделывать
> Много времени убил

Тут с опытом приходит понимание как тебе удобнее и лучше. Вообще практика mobile first удобная, но к ней привыкнуть нужно и чуть больше практики
